resources/openshift/controller-with-template.yaml

```diff
@@ -104,7 +104,7 @@ spec:
       serviceAccountName: strimzi-cluster-controller
       containers:
         - name: strimzi-cluster-controller
-          image: strimzi/cluster-controller:latest
+          image: scholzj/cluster-controller:latest
```

Comment:
scholzj ? Maybe you are using an updated cluster-controller image in your Docker Hub account but it should not be committed here.